content/blog/ajax-and-xhr-using-plain-javascript/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Ajax and XHR using plain JS"
+date: "2020-09-22"
+coverImage: "index.png"
+author: "Apeksha Gupta"
+tags: ["XHR", "JavaScript", "AJAX"]
+description: "Learn the concept of AJAX and XHR in JavaScript, the benefits of AJAX, and how easy it is to implement the AJAX into our web application."
+---
+
+## What are Ajax and XHR?
+
+**Ajax** stands for Asynchronous Javascript and  XML. Ajax is a programming technique that allows us to create dynamic, complex, and asynchronous web applications. Ajax allows us to send and receive data fro the webserver asynchronous without interfering with the current state or behavior of the web page or application.
```

Comment:
change `fro` to `from` and `asynchronous` to `asynchronously` in `Ajax allows us to send and receive data fro the webserver asynchronous without interfering with the current state or behavior of the web page or application.`

Reply:
Done